languages/js/test/parity.ts

```diff
@@ -88,8 +89,8 @@ oso.registerClass(E);
           .next()
       ).done,
       (await oso.queryRule('testOr').next()).done,
-      (await oso.queryRule('testUnifyClass', E).next()).done,
-      // oso.queryRule('testHttpAndPathMapper').next().done,
+      (await oso.queryRule('testUnifyClass', A).next()).done,
```

Comment:
this was changed by @samscott89 in #362 -- I can't remember if there was a reason for wanting to use `E` over `A`

Reply:
No reason in particular. I was just adding a few tests all using `E`.

Reply:
Was it failing with `E` or did it just seem too arbitrary @plotnick ?

Reply:
Failed.

Reply:
the Polar rule is `testUnifyClass(A);` and `E` is not an `A`